acc_test/data/invalid_kafka_user_create_duplicate.tf

```diff
@@ -0,0 +1,60 @@
+// This is part of testing "kafka user" suite, 5 of 5 
+provider "instaclustr" {
+  username = "%s"
+  api_key = "%s"
+  api_hostname = "%s"
+}
+
+resource "instaclustr_cluster" "kafka_cluster" {
+  cluster_name = "example_kafka_tf_test"
+  node_size = "%s"
+  data_centre = "US_WEST_2"
+  sla_tier = "NON_PRODUCTION"
+  cluster_network = "192.168.0.0/18"
+  wait_for_state = "RUNNING"
+  cluster_provider = {
+    name = "AWS_VPC"
+  }
+  rack_allocation = {
+    number_of_racks = 3
```

Comment:
if my above comment (consolidating tests) cant be addressed, can the number of racks reduced to mitigate the addition of test duration?